_config.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+namespace SunnysideUp\ShareThis;
+
 /**
  * developed by www.sunnysideup.co.nz
  * author: Nicolaas - modules [at] sunnysideup.co.nz
  *
  **/
 
-define('SS_SHARETHIS_DIR', 'sharethis');
+define('SS_SHARETHIS_DIR', 'resources/silverstripe-sharethis');
```

Comment:
I was seeing an error when trying to add shareThis.js as a requirement to a page, and this constant is being used to build the path. This path I think should be 'resources/sunnysideup/sharethis'.

Reply:
I got that same error after a fresh installation, maybe running `composer vendor-expose` will help, it got rid of the error in my console log :)